dolphinscheduler-ui/src/views/resource/components/resource/upload/index.tsx

```diff
@@ -59,6 +80,20 @@ export default defineComponent({
 
     const trim = getCurrentInstance()?.appContext.config.globalProperties.trim
 
+
+    watch(
+      () => props.show,
+      () => {
+        state.uploadForm.type = props.resourceType!
+        state.uploadForm.isReupload = props.isReupload
+        if (props.isReupload && props.show) {
+          state.uploadForm.id = props.id
+          state.uploadForm.name = props.name
+          state.uploadForm.description = props.description
+        }
+      }
```

Comment:
It seems that these logics are only handled when show is true.

Reply:
> It seems that these logics are only handled when show is true.

Yes, because the updateForm will be reset when the modal is hidden or the upload is confirm. If it is updated when show become false, it will cause the upload function to incorrectly refer to the re-uploaded value.